README.md

```diff
@@ -53,6 +53,34 @@ You should see output similar to the following:
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+## Running the App in a Docker Container - Development
+
+To create the image, run the command:
+```
+docker build --target development --tag todo-app:dev .
+```
+
+To run the container, run:
+```
+docker run --env-file ./.env --publish 8000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
+```
+
+Now visit [`http://localhost:8000/`](http://localhost:8000/) in your web browser to view the app.
+
+## Running the App in a Docker Container - Production
+
+To create the image, run the command:
+```
+docker build --target production --tag todo-app:prod .
+```
+
+To run the container, run:
+```
+docker run -p 8000:8000 --env-file ./.env todo-app:prod
```

Comment:
Thanks for the clear instructions! One bonus I like to add is the `-it` flag whenever running, so that I can use ctrl+c to kill the container, but it's totally a preference thing